includes/table/class-following.php

```diff
@@ -176,31 +257,56 @@ public function column_avatar( $item ) {
 	 * @param array $item Item.
 	 * @return string
 	 */
-	public function column_url( $item ) {
+	public function column_username( $item ) {
 		return sprintf(
-			'<a href="%s" target="_blank">%s</a>',
-			esc_url( $item['url'] ),
-			$item['url']
+			'<img src="%1$s" width="32" height="32" alt="%2$s" loading="lazy"/> <strong><a href="%3$s">%4$s</a></strong><br />',
+			\esc_url( $item['icon'] ),
+			\esc_attr( $item['username'] ),
+			\esc_url( $item['url'] ),
+			\esc_html( $item['post_title'] )
```

Comment:
As a follow-up, I think it would be good to add an "Undo" action link here.

Reply:
is the bulk not enough?